lib/Service/LangRopeService.php

```diff
@@ -177,6 +177,15 @@ private function requestToExApp(
 		return $finalBody;
 	}
 
+	/**
+	 * @return array<array-key, int>
+	 * @throws \RuntimeException
+	 */
+	public function getIndexedDocumentsCounts(): array {
+		$response = $this->requestToExApp('/countIndexedDocuments', 'POST');
+		return $response;
+	}
```

Comment:
we can catch a 404 here, and return `[]` to maintain compatibility, or perhaps catch the exception in the caller so an appropriate message can be shown there.

Reply:
Since 4xx doesn't  seem to throw can we just do this?

```
public function getIndexedDocumentsCounts(): array {
		$response = $this->requestToExApp('/countIndexedDocuments', 'POST');
		if (!isset($response['files__default'])) {
			return [];
		}
		return $response;
	}
```

Reply:
ah right
yeah looks good!